vector/src/main/res/layout/item_pushgateway.xml

```diff
@@ -125,6 +125,23 @@
         android:textStyle=""
         tools:text="event_id_only" />
 
+    <TextView
+        android:id="@+id/pushGatewayProfileTag"
+        style="@style/Widget.Vector.TextView.Body"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:text="@string/push_gateway_item_profile_tag"
+        android:textStyle="bold" />
+
+    <TextView
+        android:id="@+id/pushGatewayProfileTagValue"
+        style="@style/Widget.Vector.TextView.Body"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:textStyle="" />
```

Comment:
the empty text style is a little strange, would have expected `normal` instead of blank, might be worth cleaning up the file whilst we're in the area :pray:

Reply:
I toggled the text style property to bold and back to nothing selected in Android Studio and it looks like that just removes the `textStyle` properties altogether. Is that alright?

![Screenshot 2022-06-23 at 12 14 26](https://user-images.githubusercontent.com/1137962/175276088-ad38e100-76c4-499c-a0a0-8b24e2ac7701.png)

Reply:
yep, that's great! I'm a little surprised that the blank case had been compiling